ibis/backends/sql/dialects.py

```diff
@@ -336,6 +338,7 @@ def _create_sql(self, expression: sge.Create) -> str:
     return self.create_sql(expression)
 
 
+Oracle.NULL_ORDERING = "nulls_are_large"
```

Comment:
I opened the issue in sqlglot but I'm not sure if I explained it right. Feel free to chime in
https://github.com/tobymao/sqlglot/issues/3684